app/views/decidim/votings/votes/_nvotes.html.erb

```diff
@@ -10,8 +10,8 @@
   }
 </script>
 
-<div class="row">
-  <div class="column nvotes-iframe" style="height:600px;">
+<div>
+  <div class="nvotes-iframe" style="height:700px;">
```

Comment:
The booth height should adapt to browser height, it is not doing it?

Reply:
The problem is that the booth iframe has its own size, and we cannot adapt to the iframe content.